bin/program.js

```diff
@@ -68,6 +68,14 @@ function getProgram() {
       "--cookies-file <file>",
       "specifies the file name to store the persistent Cookies"
     )
+    .option(
+      "--local-storage <storage>",
+      'ability to set a local storage, key-value pairs (e.g. "bar=foo;domain=url")'
+    )
+    .option(
+      "--session-storage <storage>",
+      'ability to set a session storage, key-value pairs (e.g. "bar=foo;domain=url")'
+    )
```

Comment:
```suggestion
    .option(
      "--local-storage <values>",
      'ability to set a local storage, key-value pairs (e.g. "bar=foo;domain=url")'
    )
    .option(
      "--session-storage <values>",
      'ability to set a session storage, key-value pairs (e.g. "bar=foo;domain=url")'
    )
```

IMHO `values` says more than `storage`